### PR TITLE
OSDOCS:1968 - Updated Requirements for using your VPC with links to AWS doc

### PR DESCRIPTION
**Pull request by bmcelvee**
Manual cherrypick of https://github.com/openshift/openshift-docs/pull/38264.

### CODE REVIEW

File: modules/installation-custom-aws-vpc.adoc [hunk 1]
```diff
@@ -4,6 +4,13 @@
 // * installing/installing_aws/installing-aws-private.adoc
 // * installing/installing_aws/installing-aws-vpc.adoc
 
+ifeval::["{context}" == "installing-aws-china-region"]
+:aws-china:
+endif::[]
```
Comment (mjpytlak) [inline modules/installation-custom-aws-vpc.adoc:9]: @bmcelvee The `ifeval` for aws china does not apply to 4.8. Please remove it and the closing statement.

Comment (bmcelvee) [reply modules/installation-custom-aws-vpc.adoc]: Done!

**Pull request merged**